packages/boutique/backend/package.json

```diff
@@ -2,14 +2,15 @@
   "name": "@boutique/backend",
   "scripts": {
     "start": "node dist/index.js",
-    "build:deps": "pnpm --filter @shared/backend build",
+    "build:deps": "pnpm --filter @shared/backend build && pnpm --filter @shared/boutique build",
     "build": "pnpm build:deps && tsc --build tsconfig.build.json && tsc-alias -p tsconfig.build.json",
     "test": "jest --passWithNoTests --maxWorkers=2"
   },
   "dependencies": {
     "@google-cloud/logging-winston": "^6.0.0",
     "@interledger/open-payments": "^6.8.2",
     "@shared/backend": "workspace:*",
+    "@shared/boutique": "workspace:*",
```

Comment:
as it is only for boutique, it make sense to be part of boutique `boutique/shared`